src/elm/Pages/MyAccount/View.elm

```diff
@@ -20,12 +20,28 @@ view user =
 
                 _ ->
                     ""
+        login =
```

Comment:
since you meed to case the user twice, you can do:

```elm
  (login, avatar) -> 
     case user of
         Success val -> ( val.login, img [ src user'.avatarUrl ] )
        _ -> ( "", div [] [])
```